relay-event-normalization/src/normalize/mod.rs

```diff
@@ -257,72 +241,35 @@ pub struct ModelCosts {
     /// The version of the model cost struct
     pub version: u16,
 
-    /// The mappings of model ID => cost (used in version 1)
-    #[serde(default, skip_serializing_if = "Vec::is_empty")]
-    pub costs: Vec<ModelCost>,
-
-    /// The mappings of model ID => cost as a dictionary (version 2)
+    /// The mappings of model ID => cost as a dictionary
     #[serde(default, skip_serializing_if = "HashMap::is_empty")]
     pub models: HashMap<String, ModelCostV2>,
 }
 
 impl ModelCosts {
-    const MAX_SUPPORTED_VERSION: u16 = 2;
+    const SUPPORTED_VERSION: u16 = 2;
 
     /// `true` if the model costs are empty and the version is supported.
     pub fn is_empty(&self) -> bool {
-        (self.costs.is_empty() && self.models.is_empty()) || !self.is_enabled()
+        self.models.is_empty() || !self.is_enabled()
     }
 
     /// `false` if measurement and metrics extraction should be skipped.
     pub fn is_enabled(&self) -> bool {
-        self.version > 0 && self.version <= ModelCosts::MAX_SUPPORTED_VERSION
+        self.version == 2
     }
 
     /// Gets the cost per token, if defined for the given model.
     pub fn cost_per_token(&self, model_id: &str) -> Option<ModelCostV2> {
-        match self.version {
-            1 => {
-                let input_cost = self.costs.iter().find(|cost| cost.matches(model_id, false));
-                let output_cost = self.costs.iter().find(|cost| cost.matches(model_id, true));
-
-                // V1 costs were defined per 1k tokens, so we need to convert to per token.
-                if input_cost.is_some() || output_cost.is_some() {
-                    Some(ModelCostV2 {
-                        input_per_token: input_cost.map_or(0.0, |c| c.cost_per_1k_tokens / 1000.0),
-                        output_per_token: output_cost
-                            .map_or(0.0, |c| c.cost_per_1k_tokens / 1000.0),
-                        output_reasoning_per_token: 0.0, // in v1 this info is not available
-                        input_cached_per_token: 0.0,     // in v1 this info is not available
-                    })
-                } else {
-                    None
-                }
-            }
-            2 => self.models.get(model_id).copied(),
-            _ => None,
+        if self.version == Self::SUPPORTED_VERSION {
```

Comment:
In combination with the suggestion above:
```suggestion
        if self.is_enabled() {
```

Reply:
Yep, thanks!